src/HPolyhedron.jl

```diff
@@ -476,8 +496,40 @@ FAQ](https://www.cs.mcgill.ca/~fukuda/soft/polyfaq/node24.html).
 """
 function remove_redundant_constraints!(P::PT;
                                        backend=GLPKSolverLP())::Bool where {N, PT<:HPoly{N}}
+    remove_redundant_constraints!(P.constraints, backend=backend)
```

Comment:
Done. Feels like such a function need not live in this `AbstractPolytope`, since it does not make use of it. I think that we should either restrict it to an abstract polytope but live with the consequences (having to add more functions for more sets, which can be ok since it lets to know that "those methods will work"), or move this function to `LazySets.jl` as a fallback implementation for any lazyset (actual and future).

Reply:
Then let us move it.

Reply:
Do we want to move `tosimplehrep` to another file? Maybe not `helper_functions` because we export it. From my side it can also stay and we eventually pull it up to `AbstractPolyhedron` if that comes.

Reply:
yes to stay or move later to the abstract hpolyhedron. i don;t like `helper_functions` to grow as a mix of everything